lnwallet/commitment.go

```diff
@@ -177,6 +185,21 @@ func DeriveCommitmentKeys(commitPoint *btcec.PublicKey,
 		)
 	}
 
+	// If this is a taproot commitment, then we'll use the funding keys to
+	// generate a combined funding key.
+	if chanType.IsTaproot() {
```

Comment:
Here you could also consider making the legacy channel type the exception (`if !isTaproot`).

Reply:
Impact here seems minimal as this adds a field that wouldn't exist otherwise, so not like we're able to get rid of extra indentation or anything like that.